### ISSUE TITLE
Support of sessions

### ISSUE THREAD

**Issue by GoogleCodeExporter** [otros-systems/otroslogviewer#262]
```
What steps will reproduce the problem?
1. Open otroslogviewer
2. Select some log-files to view or tail
3. Close otroslogviewer
4. Re-Open otroslogviewer
5. I have to open all log-files again 

What is the expected output? What do you see instead?
Either the logviewer shall save a "session-state" and try to re-open all
logfiles on re-opening the application or there should be a possibility
to explicit save the "session-state" (all open files, parsers etc).

What version of the product are you using? On what operating system?
2013-01-24 on Windows

Please provide any additional information below.
For example "LogExpert" re-opens all files from a previous session.
```

Original issue reported on code.google.com by `roland.s...@gmail.com` on 17 Jul 2013 at 1:33

**Comment by pnowy** [comment]
It would be great to have that option. Unfortunately currently after computer restart I have to spend some time to load all sessions and change the session names because the names of tabs have names likes 'Multiple logs' not the session name.

**Comment by otrebski** [comment]
Issue is already implemented in 1.4.6

**Comment by pnowy** [comment]
@otrebski are you sure that was implemented? Every time when I open the tool I have to manually load the sessions (the previous sessions/files are not opened automatically). Version which I'm using 1.4.6 on Windows 10.

![image](https://cloud.githubusercontent.com/assets/3254609/25694989/9295f314-30b1-11e7-8f4d-d81e61e5c10c.png)
